src/VirtoCommerce.Platform.Core/Security/ApplicationUser.cs

```diff
@@ -87,11 +87,6 @@ public virtual void Patch(ApplicationUser target)
             target.Status = Status;
             target.Password = Password;
             target.PasswordExpired = PasswordExpired;
```

Comment:
You can't remove these lines because during import or update a user through API there also might be passed the user roles along with modified user object. Without this code, the roles will be ignored and won't be updated in the destination system.

Reply:
I think it's a redundant code. cause: call CustomUserManager.CrearteAsyn/UpdateAsync when try to import/update some users. there are no create/update roles in  the methods(in source code). look at
https://github.com/dotnet/aspnetcore/blob/release/3.1/src/Identity/Extensions.Core/src/UserManager.cs#L468
https://github.com/dotnet/aspnetcore/blob/release/3.1/src/Identity/EntityFrameworkCore/src/UserOnlyStore.cs#L156
after calling a base updateAsync
https://github.com/VirtoCommerce/vc-platform/blob/dev/src/VirtoCommerce.Platform.Web/Security/CustomUserManager.cs#L175
there are not records in `AspNetUserRoles`

Reply:
Ok, make sense.